### PR TITLE
[SPARK-39761][K8S][DOCS] Add Apache Spark images info in running-on-kubernetes doc

### PR DESCRIPTION
**Pull request by Yikun**
### What changes were proposed in this pull request?
Add Apache Spark images info in running-on-kubernetes doc

### Why are the changes needed?
We add the docker image in dockerhub for spark relese like: [v3.1.3,v3.2.1,v3.3.0](https://hub.docker.com/r/apache/spark/tags) and release steps on https://github.com/apache/spark-website/pull/400

It's better to add note it in running-on-kubernetes doc.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
SKIP_API=1 bundle exec jekyll serve --watch
image

### CODE REVIEW

**Comment by Yikun** [comment]
cc @zhengruifeng @dongjoon-hyun @holdenk

File: docs/running-on-kubernetes.md [hunk 1]
```diff
@@ -113,6 +113,8 @@ $ ./bin/docker-image-tool.sh -r <repo> -t my-tag -p ./kubernetes/dockerfiles/spa
 $ ./bin/docker-image-tool.sh -r <repo> -t my-tag -R ./kubernetes/dockerfiles/spark/bindings/R/Dockerfile build
 ```
 
+You can also use the [Apache Spark docker images](https://hub.docker.com/r/apache/spark) (such as `apache/spark:<version>`) directly.
```
[the remark this answers is not in the document]
Comment (Yikun) [reply docs/running-on-kubernetes.md]: oh my bad sorry, addressed

**Review by zhengruifeng — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
@xinrong-meng wanna try merging this one?

**Review by martin-g — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. (Leave this open to new committer as Hyukjin commented already). 
:)

**Comment by xinrong-meng** [comment]
Merged to master. Thanks!

**Pull request closed without merging**